src/_zkapauthorizer/replicate.py

```diff
@@ -113,8 +132,8 @@ class EventStream:
         to the schema will be accompanied with an increment to this value.
     """
 
-    changes: tuple[Change, ...]
-    version: int = 1
+    changes: Sequence[Change] = field(converter=tuple)
+    version: ClassVar[int] = 2
```

Comment:
It could be since we're not promising compatibility between current main@HEAD and this branch.  I didn't even think about it very much, I just bumped it because the schema did change (and I think having extra versions doesn't ever hurt).

Reply:
Well, sure ... but also it's weird that the only version that'll be out there is "2" (because if you _do_ give it a "1" you'll get an error anyway)